__test__/ResponsiveCardsRow.test.js

```diff
@@ -15,10 +15,16 @@ describe(`ResponsiveCardsRow`, () => {
         batmanFilmsSpeciesCardProps,
         findingNemoSpeciesCardProps,
         theSmithHouseholdImageCardProps
-      ]
+      ],
+    CardClass: () => `topClass`
   }
 
-  test(`matches snapshot`, () => {
-    expect(mount(<ResponsiveCardsRow {...props}/>)).toMatchSnapshot()
+  test(`displays cards in default row mode`, () => {
+    const wrapper = shallow(<ResponsiveCardsRow {...props}/>)
+    expect(wrapper.find(`#slide`)).toHaveLength(0)
```

Comment:
This test doesn’t make sense, since `ResponsiveCardsRow` doesn’t know what `#slide` is. Imagine if we decided to check all the IDs or classes this component does **not** have.